overcast/runner/__init__.py

```diff
@@ -373,7 +415,7 @@ def wait():
 
     def shell_step_cmd(self, details, env_prefix=''):
         if details.get('type', None) == 'remote':
-             node = self.nodes[details['node']][1]
+              node = next(iter(self.nodes[details['node']].fip_addresses))
```

Comment:
Well, the code supports creating it, but it's never used. Everywhere we reference node.fip_addresses, we just pick one at random, so there's really not much point in keeping track of them.